### ISSUE TITLE
User history 

### ISSUE THREAD

**Issue by satenikmnatsakanyan-aua** [artur787/Dice_Game#1]
@artur787 you were going to save the user game data in the file right?

**Comment by artur787** [comment]
> @artur787 you were going to save the user game data in the file right?
Yes, the usernames, passwords and their scores for the leaderboard are going to be saved in a separate .json file. I will do that part of my work during the upcoming week
